src/components/dialog/Dialog.stories.mdx

```diff
@@ -634,6 +634,95 @@ import PageHeader from 'blocks/PageHeader';
   </Story>
 </Canvas>
 
+### Switching modals
```

Comment:
For testing purposes, To be removed

Reply:
Actually, maybe it should stay here as an example? @coderitual wdyt? Do we need it?